File: Data Structures & Algorithms/top-k-elements-in-list/submission-1.py
class Solution:
    def topKFrequent(self, nums: List[int], k: int) -> List[int]:
        n = []
        freq = []
        retList = []
        nums.sort()
        i = 0
        j = i + 1

        while(i < len(nums)):
            while(j < len(nums) and nums[i] == nums[j]):
                j += 1
            
        
            n.append(nums[i])
            freq.append(j-i) 
            i = j

        for i in range(k):
            retList.append(n[freq.index(max(freq))])
            del n[freq.index(max(freq))]
            freq.remove(max(freq))

        return retList

File: Data Structures & Algorithms/remove-duplicates-from-sorted-array/submission-1.java
class Solution {
    public int removeDuplicates(int[] nums) {
          int l = 0;
          int r = 1;
          int k = 0;
          if(nums.length == 1) return 1;
          
          while(l < r && r < nums.length){
               if(nums[l] == nums[r]){
                    l = r;
                    r += 1;
               }else{
                    nums[k] = nums[l];
                    l = r;
                    r += 1;
                    k++;
               }
          }
          if(nums[l] != nums[k-1]){
               nums[k] = nums[l];
               k++;
          }
          return k;
    }
}